From: Jeff Johnson <quic_jjohnson@quicinc.com>
Subject: kunit/fortify: add missing MODULE_DESCRIPTION() macros
Date: Fri, 31 May 2024 15:45:22 -0700

make allmodconfig && make W=1 C=1 reports:
WARNING: modpost: missing MODULE_DESCRIPTION() in lib/memcpy_kunit.o
WARNING: modpost: missing MODULE_DESCRIPTION() in lib/fortify_kunit.o

Add the missing invocations of the MODULE_DESCRIPTION() macro.

Link: https://lkml.kernel.org/r/20240531-md-lib-fortify_source-v1-1-2c37f7fbaafc@quicinc.com
Signed-off-by: Jeff Johnson <quic_jjohnson@quicinc.com>
Cc: Kees Cook <keescook@chromium.org>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 lib/fortify_kunit.c |    1 +
 lib/memcpy_kunit.c  |    1 +
 2 files changed, 2 insertions(+)

--- a/lib/fortify_kunit.c~kunit-fortify-add-missing-module_description-macros
+++ a/lib/fortify_kunit.c
@@ -1096,4 +1096,5 @@ static struct kunit_suite fortify_test_s
 
 kunit_test_suite(fortify_test_suite);
 
+MODULE_DESCRIPTION("Runtime test cases for CONFIG_FORTIFY_SOURCE");
 MODULE_LICENSE("GPL");
--- a/lib/memcpy_kunit.c~kunit-fortify-add-missing-module_description-macros
+++ a/lib/memcpy_kunit.c
@@ -510,4 +510,5 @@ static struct kunit_suite memcpy_test_su
 
 kunit_test_suite(memcpy_test_suite);
 
+MODULE_DESCRIPTION("test cases for memcpy(), memmove(), and memset()");
 MODULE_LICENSE("GPL");
_
